dependencies are not updated

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Explain what you did

i wanted to update both the themed and base packages to 4.0.0-rc.8 and got an error. 

on this line the dependencies were not updated:
https://github.com/react-native-elements/react-native-elements/blob/next/packages/themed/package.json#L53-L56


### Expected behavior

updating should work

### Describe the bug

`npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @rneui/themed@4.0.0-rc.8
npm ERR! Found: @rneui/base@4.0.0-rc.8
npm ERR! node_modules/@rneui/base
npm ERR!   @rneui/base@"4.0.0-rc.8" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @rneui/base@"4.0.0-rc.7" from @rneui/themed@4.0.0-rc.8
npm ERR! node_modules/@rneui/themed
npm ERR!   @rneui/themed@"4.0.0-rc.8" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @rneui/base@4.0.0-rc.7
npm ERR! node_modules/@rneui/base
npm ERR!   peer @rneui/base@"4.0.0-rc.7" from @rneui/themed@4.0.0-rc.8
npm ERR!   node_modules/@rneui/themed
npm ERR!     @rneui/themed@"4.0.0-rc.8" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.`

### Steps To Reproduce

```Markdown
trying to use base and themed together doesn't work on the latest versions
```


### Screenshots

_No response_

### Your Environment

_No response_

--- a/packages/themed/package.json
+++ b/packages/themed/package.json
@@ -50,10 +50,10 @@
     "logo": "https://opencollective.com/react-native-elements/logo.txt"
   },
   "peerDependencies": {
-    "@rneui/base": "4.0.0-rc.7"
+    "@rneui/base": "*"
   },
   "devDependencies": {
-    "@rneui/base": "4.0.0-rc.7"
+    "@rneui/base": "latest"
   },
   "repository": {
     "type": "git",

